tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -476,9 +520,9 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id, HR
     if (Logger::IsDebugEnabled())
     {
         Logger::Debug("ModuleLoadFinished: ", module_id, " ", module_info.assembly.name, " AppDomain ",
-                      module_info.assembly.app_domain_id, " ", module_info.assembly.app_domain_name, std::boolalpha,
-                      " | IsNGEN = ", module_info.IsNGEN(), " | IsDynamic = ", module_info.IsDynamic(),
-                      " | IsResource = ", module_info.IsResource(), std::noboolalpha);
+                        module_info.assembly.app_domain_id, " ", module_info.assembly.app_domain_name, std::boolalpha,
```

Comment:
There's a lot of little whitespace changes in this file. I wonder if your IDE is not loading the `.clang-format`.

Reply:
Hmm, I'm not sure why that is. I'll follow up with some more commits to undo the whitespace changes before merging